ui/app/pages/settings/alerts-tab/alerts-tab.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { I18nContext } from '../../../contexts/i18n'
+import Tooltip from '../../../components/ui/tooltip-v2'
+import ToggleButton from '../../../components/ui/toggle-button'
+import { setAlertEnabledness } from '../../../store/actions'
+import { getAlertEnabledness } from '../../../ducks/metamask/metamask'
+
+const AlertSettingsEntry = ({ alertId, description, title }) => {
+  const t = useContext(I18nContext)
+  const dispatch = useDispatch()
+  const isEnabled = useSelector((state) => getAlertEnabledness(state)[alertId])
+
+  return (
+    <>
+      <span>
+        { title }
+      </span>
+      <Tooltip
+        position="top"
+        title={description}
+        wrapperClassName="alerts-tab__description"
+      >
+        <i className="fa fa-info-circle" />
+      </Tooltip>
+      <ToggleButton
+        offLabel={t('off')}
+        onLabel={t('on')}
+        onToggle={() => dispatch(setAlertEnabledness(alertId, !isEnabled))}
+        value={isEnabled}
+      />
+    </>
+  )
+}
+
+AlertSettingsEntry.propTypes = {
+  alertId: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+}
+
+const AlertsTab = () => {
+  const t = useContext(I18nContext)
+
+  const alertConfig = {
+    unconnectedAccount: {
```

Comment:
Can we put these keys in a single location? Currently this needs to match with the string we're using in the selector and that seems like it might be error-prone. Could we put the keys in the bg?

Reply:
Both of:

- `ui/app/ducks/metamask/metamask.js:373`
- `ui/app/ducks/alerts/unconnected-account.js:20`

Reply:
Yeah, we can do that. I had considered it before - there was one revision where I exported the `name` variable in `ui/app/ducks/alerts/unconnected-account.js` and used that in a few places. It felt a bit tedious at the time, but now that it's in the background too it probably makes sense to add it as a constant somewhere.

Reply:
This was done in 5c4df5538361d5238f4dc3100c47acc8f83e4db8